Makefile

```diff
@@ -109,15 +149,37 @@ run-static-analisys:
 run-unit-test:
 	echo TODO: $(@)
 
-run-smoke-test:
-	echo TODO: $(@)
+run-contract:
+	sed -i -e 's|SECRET_TO_REPLACE|$(COGNITO_POSTCODE_AUTH)|g' $(APPLICATION_TEST_DIR)/contract/environment/postcode_contract.postman_environment.json
+	make stop
+	make start PROFILE=local
+	make docker-run-postman \
+		DIR="$(APPLICATION_TEST_DIR)/contract" \
+		CMD=" \
+			run PostcodeAPIContractTests.postman_collection.json -e environment/postcode_contract.postman_environment.json --verbose --insecure \
+		"
+	make project-stop
+
+run-smoke:
+	sed -i -e 's|SECRET_TO_REPLACE|$(COGNITO_POSTCODE_AUTH)|g' $(APPLICATION_TEST_DIR)/contract/environment/postcode_smoke.postman_environment.json
+	make restart
+	make docker-run-postman \
+		DIR="$(APPLICATION_TEST_DIR)/contract" \
+		CMD=" \
+			run PostcodeAPISmokeTests.postman_collection.json -e environment/postcode_smoke.postman_environment.json --verbose --insecure \
+		"
+	make project-stop
+
+project-aws-get-admin-secret:
```

Comment:
There is already a make target that will do this. Have a look in aws.mk file. Maybe aws-secret-get will do this for you